### ISSUE TITLE
Add browser compatibility info (especially IE)

### ISSUE THREAD

**Issue by mh-nichts** [jimmynotjim/scrollnav#90]
Hello,
Thank you for this plugin.
I open this issue because I can't find a precise information about the browser compatibility : I currently have a bug with IE11 but I don't know if that browser is normally supported and thus deserves opening an issue...
It would help to have the browser compatibility info in the README.

**Comment by jimmynotjim** [comment]
Thanks for the feedback and I definitely agree, I will add a browser list to the list of upcoming changes.

Any chance you can recreate the error in an environment I can test, like [Codepen](https://codepen.io/) or [JSFiddle](https://jsfiddle.net/). The current release of scrollNav is quite old and I'm working on a complete rewrite, but it's still a few weeks from release. Hopefully we can figure out what this issue is in the meantime.

**Comment by mh-nichts** [comment]
Thank you for the quick response !

While preparing the reduced test case, I found that I couldn't reproduce the error in Codepen.

So I double-checked my project, and it happens that the error is very specific : when I append "#" at the URL of the page where I use scrollNav, I have a jQuery error on load : 
"Syntax error, unrecognized expression: #"
This seems to happen after the onInit callback, but then causes the onRender callback not working.

Here is my test case on Codepen, even if I can't get the error there, but you can see the structure and situation : 
https://codepen.io/mh-nichts/full/pVoNaP/
I guess I can't reproduce the exact same case because of the way Codepen is dealing with hashes (so putting "#" at the end of the URL doesn't work the same as usual).

Basically, if you would test it outside of Codepen : there is a console.log on init and on render.
They work fine in a normal situation, but on IE11 only, and only when there is "#" appended to the URL, onInit is called, then you get the jQuery error, and then it never goes until the onRender callback. 
The scrollNav is created though, and works.

I should specify that I'm sure it's a scrollNav issue because it only happens on the pages where I use it, and if I comment the scrollNav initialization, I don't have any error.

Thanks in advance for investigating it !

**Comment by jimmynotjim** [comment]
@mh-nichts Thanks setting up that demo. Sorry I couldn't look sooner, had a busy end to the week. Codepen offers a debug mode which shouldn't take over hash handling. I've tested this in IE 11 on a Win 10 box in SauceLabs and it seems to be working correctly. 

Could you double check this url in your environment that's failing and let me know what your exact environment setup is? 

https://s.codepen.io/jimmynotjim/debug/RyWPxJ/ZoABajgWJyYr#scrollNav-2

<img width="1680" alt="screen shot 2018-04-21 at 9 57 20 am" src="https://user-images.githubusercontent.com/1280430/39085584-7ed03fa2-454a-11e8-870a-3abe1db7c941.png">

I'm still working on the v3 release of scrollNav, but I'm hoping to have it released soon. How comfortable are you with using native JS rather than jQuery? Would you be interested in testing it?

**Comment by jimmynotjim** [comment]
@mh-nichts Were you able to take a look at this and verify your environment fails with my Codepen example?

**Comment by jimmynotjim** [comment]
Browser compatibility documentation has been adding via links to caniuse.com. Closing.

**Comment by mh-nichts** [comment]
Sorry for replying so late (one year later !). 

When I finally had time to come back on this issue I couldn't test your codepen, I guess it's because the "debug" view had already expired.

Now I come back with some news : on a new project, where we still use the 2.7.3 scrollNav version (with jQuery), it now works correctly on the very same IE11, # in the URL or not. 
I can't say what solved the problem but it might be that my other project had something else messing with it.

Anyway I want to thank you for documenting the browser compatibility in the new versions !

**Comment by jimmynotjim** [comment]
@mh-nichts Nice, glad you got it working. It's always hard to test issues like this and sandboxing it at Codepen or another similar service is a great way at debugging. If you run into any issues with v2 feel free to reach out.
